trollmoves/server.py

```diff
@@ -765,7 +770,7 @@ class CTimer(Thread):
     t.cancel() # stop the timer's action if it's still waiting
     """
 
-    def __init__(self, interval, function, args=[], kwargs={}):
+    def __init__(self, interval, function, *args, **kwargs):
```

Comment:
This change is not correct, args has to be passed as a list, kwargs as a dict as per the Thread call signature

Reply:
Oh, right. Thought those original versions were considered "dangerous" and didn't think of the use-case at all. Good catch!